### PR TITLE
upgrade playbook

### PR DESCRIPTION
**Pull request by sdodson**
rough cut of documenting the upgrade playbook, the playbook hasn't merged upstream yet so this may slip to tomorrow

### CODE REVIEW

**Comment by adellape** [comment]
:100: x 10 = PR 1000!!!

**Comment by sdodson** [comment]
@adellape probably as good as i'm going to get it, I struggled with how to differentiate between manual and automated playbook while still ensuring that the version specific steps remain in the TOC, this is the best I could come up with.

File: install_config/upgrades.adoc [hunk 1]
```diff
@@ -164,12 +178,12 @@ routing service].
...
 ifdef::openshift-origin[]
 OpenShift Origin 1.0.4 or 1.0.5, first see the
-link:#additional-instructions-per-release[Additional Instructions per Release]
+link:#additional-instructions-per-release[Additional Manual Instructions per Release]
 section for important steps specific to your upgrade,
 endif::[]
 ifdef::openshift-enterprise[]
 OpenShift Enterprise 3.0.1.0 or 3.0.2.0, first see the
-link:#additional-instructions-per-release[Additional Instructions per Release]
+link:#additional-instructions-per-release[Additional Mannual Instructions per Release]
```
Comment (adellape) [inline install_config/upgrades.adoc]: s/Mannual/Manual/

File: install_config/upgrades.adoc [hunk 2]
```diff
@@ -597,3 +611,66 @@ networkConfig:
...
+available you may use the upgrade playbook to automate the upgrade process. This
+playbook will perform the following steps for you:
+
+ * Applies latest configuration by re-running the installation playbook
+ * Upgrade and restart master services
+ * Upgrade and restart node services
+ * Applies the latest cluster policies
+ * Updates the default router if one exists
+ * Updates the default registry if one exists
+ * Updates image streams and quickstarts
```
Comment (adellape) [inline install_config/upgrades.adoc]: s/quickstarts/InstantApp templates/

**Comment by adellape** [comment]
@sdodson Couple minor things inline. I noticed the last two example code output blocks aren't conditionalized for Origin vs OSE, btw. E.g., `openshift3/ose-docker-registry:v3.0.2.0` and `version v3.0.2.0-6-g863e73a`.

**Pull request merged**